app/views/layouts/beta/_footer_content.html.erb

```diff
@@ -1,44 +1,41 @@
-<footer class="govuk-footer " role="contentinfo">
```

Comment:
As the beta code will be going away as soon as we switch to the new design, I wouldn't expect to see any changes here (unless this is being used elsewhere?